<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LANChatClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="userListGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>334</width>
        <height>151</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>334</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Список пользователей</string>
      </property>
      <layout class="QGridLayout" name="userGridLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="userListWidget">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>306</width>
           <height>131</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>306</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="authGroupBox">
      <property name="minimumSize">
       <size>
        <width>334</width>
        <height>191</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>334</width>
        <height>191</height>
       </size>
      </property>
      <property name="title">
       <string>Авторизация</string>
      </property>
      <widget class="QLabel" name="authLoginLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="authLoginLineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>246</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>246</width>
         <height>31</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="authClientIPv4TextLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Адрес клиента:</string>
       </property>
      </widget>
      <widget class="QLabel" name="authClientIPv4ValueLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>176</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>176</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Нет данных</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="authServerIPv4TextLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Адрес сервера:</string>
       </property>
      </widget>
      <widget class="QLabel" name="authServerIPv4ValueLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>176</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>176</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Нет данных</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="authSignInPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>141</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>321</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Подключиться к серверу</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QGroupBox" name="chatGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>441</width>
        <height>581</height>
       </size>
      </property>
      <property name="title">
       <string>Чат</string>
      </property>
      <layout class="QGridLayout" name="chatGridLayout">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="chatPlainTextEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>414</width>
           <height>527</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="actionsGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>331</width>
        <height>191</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>331</width>
        <height>201</height>
       </size>
      </property>
      <property name="title">
       <string>Операции пользователя</string>
      </property>
      <widget class="QPlainTextEdit" name="actionsSendMessagePlainTextEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>314</width>
         <height>61</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>51</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="actionsSendMessagePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>314</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Отправить сообщение</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionsSendFilePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>314</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Отправить файл</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
